rules/windows/builtin/win_mal_service_installs.yml

```
@@ -1,9 +1,13 @@
 title: Malicious Service Installations
 id: 2cfe636e-317a-4bee-9f2c-1066d9f54d1a
-description: Detects known malicious service installs that only appear in cases of lateral movement, credential dumping and other suspicious activity
+description: Detects known malicious service installs that only appear in cases of lateral movement, credential dumping, and other suspicious activities.
 author: Florian Roth, Daniil Yugoslavskiy, oscd.community (update)
 date: 2017/03/27
 modified: 2021/05/27
```

As detection is update pls change `modified` too
```suggestion
modified: 2021/07/06
```